Tests/ArmoniK.EndToEndTests/ArmoniK.EndToEndTests.Client/Tests/LargeSubmitAsync/LargeSubmitAsyncClient.cs

```diff
@@ -224,18 +225,21 @@ public override void EntryPoint()
                                                                                                       ParamsHelper(numbers,
                                                                                                                    workloadTimeInMs),
                                                                                                       this,
-                                                                                                      token);
+                                                                                                      token: token);
 
                                                                return result;
                                                              });
 
                            foreach (var task in tasksInBucket)
                            {
-                             resultTask.Add(task.Result);
+                             resultTask.Add(task);
                            }
                          });
 
-    return resultTask.ToList();
+    resultTask.AsParallel()
+              .ForAll(task => results.Add(task.Result));
+
+    return results.ToList();
```

Comment:
What do this has to do with TaskOptions?
Besides, you could most likely use `.Select` instead of `.ForEach` that adds into a concurrentbag

Reply:
File doesn't exist anymore